packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -168,25 +170,35 @@ abstract class InAppPurchaseConnection {
   Future<bool> buyConsumable(
       {@required PurchaseParam purchaseParam, bool autoConsume = true});
 
-  /// (App Store only) Mark that purchased content has been delivered to the
+  /// Mark that purchased content has been delivered to the
   /// user.
   ///
   /// You are responsible for completing every [PurchaseDetails] whose
-  /// [PurchaseDetails.status] is [PurchaseStatus.purchased] or
-  /// [[PurchaseStatus.error]. Completing a [PurchaseStatus.pending] purchase
-  /// will cause an exception.
-  ///
-  /// This throws an [UnsupportedError] on Android.
-  Future<void> completePurchase(PurchaseDetails purchase);
+  /// [PurchaseDetails.status] is [PurchaseStatus.purchased].
+  /// Additionally, the purchase needs to be completed if the [PurchaseDetails.status]
+  /// [[PurchaseStatus.error].
```

Comment:
subjective, nit: I think the previous sentence was a little more clear here than the current one.

Reply:
actually, since the purchase should only be completed by android when the status is purchased, I tried to rephrased a bit. But the sentence some how got the broken.
I meant to say:
Additionally on iOS, the purchase needs to be completed if the [PurchaseDetails.status] is [PurchaseStatus.error].